packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -478,11 +482,88 @@ function useProcessMedia({
     ]
   );
 
+  /**
+   * Crop hidden video using FFmpeg.
+   *
+   * @param {import('@googleforcreators/media').Resource} resource Resource object.
+   */
+  const cropHiddenVideo = useCallback(
+    ({ id: elementId, resource: oldResource }, cropParams) => {
+      const { id: resourceId, src: url, mimeType } = oldResource;
+      const { newWidth, newHeight, cropElement } = cropParams;
+
+      const onUploadSuccess = ({ id, resource }) => {
+        copyResourceData({ oldResource, resource });
+        updateOldTranscodedObject(oldResource.id, resource.id, 'source-image');
+        deleteMediaElement({ id: oldResource.id });
```

Comment:
<!-- general note -->
**📝 note:**

If the original resource is deleted and the user clicks "Undo", and then tries to crop again, I assume it would cause errors since the original resource doesn't exist anymore? Since the user can undo the action then not sure if we can permanently delete the original resource 🤔

Reply:
Perhaps in this case the original resource should be kept - most of that part of the code was copied over from the trim functionality.

Reply:
removed the delete